packages/sdk/src/session/index.ts

```diff
@@ -0,0 +1,28 @@
+import { createStore } from "./../store/composed";
+
+export interface Currency {
+  code: string; // Ex: USD
+  symbol: string; // Ex: $
+}
+
+export interface Person {
+  id: string;
+  email: string;
+  givenName: string;
+  familyName: string;
+}
+
+export interface Session {
+  locale: string; // en-US
+  currency: Currency;
+  country: string; // BRA
+  channel: string | null;
+  postalCode: string | null;
+  person: Person | null;
```

Comment:
Why `null` instead `undefined`?
```suggestion
  channel?: string;
  postalCode?: string;
  person?: Person;
```

Reply:
same in here